.github/workflows/docs-only-makecheck.yml

```diff
@@ -0,0 +1,33 @@
+# The 'make check' check is a required check for all Pull Requests.
+# However, we don't need to run the Jenkins job if a PR only changes
+# documentation. This GitHub Action will create a dummy 'make check'
+# check in lieu of the Jenkins job running and exiting 0 to satisfy the check.
+name: make check
+on:
+  pull_request:
+    paths-ignore:
```

Comment:
I guess you have tested this and it worked, but [Github docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-and-excluding-paths) say you cannot mix paths and paths-ignore in the same event of a workflow.

Reply:
I'm not sure it's possible to test these without merging them?  I missed that part though.  I guess we'll have to pick one and use `!`